frontend/lib/src/hocs/withFullScreenWrapper/withFullScreenWrapper.tsx

```diff
@@ -19,7 +19,7 @@ import hoistNonReactStatics from "hoist-non-react-statics"
 
 import FullScreenWrapper from "@streamlit/lib/src/components/shared/FullScreenWrapper"
 
-interface Props {
+export interface Props {
```

Comment:
I was getting some typescript error where if I added `isFullScreen` to the props, 
```
Default export of the module has or is using private name 'Props'.
```
As a result, just exported this and that fixed it.